website/addons/zotero/tests/test_models.py

```diff
@@ -10,8 +10,7 @@
 from tests.factories import UserFactory, ProjectFactory
 from website.addons.zotero.tests.factories import (
     ZoteroAccountFactory,
-    ZoteroUserSettingsFactory,
-    ExternalAccountFactory,
+    ZoteroUserSettingsFactory
```

Comment:
`ZoteroUserSettingsFactory` is no longer being used in this file and can be removed